### DISCUSSION TITLE
Connection between two genes

### DISCUSSION THREAD

**Discussion by Selbi22** [chanzuckerberg/single-cell-data-portal#2962] [Q&A] [answered]
Hi!

I am an international student at Montana State University Billings. I would like to ask you some questions if possible regarding how to use spinalcordatlas.org

I wonder if there is a website or a video about how the website works?

I am interested in cholinergic neurons and I am trying to see the connection between two genes. What functions I could use to see the connection?

I wonder if there is a website or a video about how the website works?

Thank you very much in advance!

**Comment by BAevermann** [comment, marked as answer]
Hello Selbi22,

The website at spinalcordatlas.org is not supported by the cellxgene team. They are self-hosting using an independent cellxgene instance, so any questions should be directed to their support team. I believe they can be reached at "Direct correspondence to jablum(at)stanford(dot)edu". Good luck!

Thanks,

Brian Aevermann
